kubelet/metadata.csv

```diff
@@ -63,3 +63,6 @@ kubernetes.kubelet.memory.rss,gauge,,byte,,Size of kubelet RSS in bytes,-1,kuber
 kubernetes.runtime.cpu.usage,gauge,,nanocore,,The number of cores used by the runtime,-1,kubernetes,k8s.runtime.cpu
 kubernetes.runtime.memory.rss,gauge,,byte,,Size of runtime RSS in bytes,-1,kubernetes,k8s.runtime.mem.rss
 kubernetes.kubelet.container.log_filesystem.used_bytes,gauge,,byte,,Bytes used by the container's logs on the filesystem (requires kubernetes 1.14+),0,kubernetes,k8s.kubelet.container.log_filesystem
+kubernetes.kubelet.pod.start.duration,gauge,,microseconds,,Duration in microseconds for a single pod to go from pending to running,0,kubernetes,k8s.kubelet.pod.start.duration
+kubernetes.kubelet.pod.worker.duration,gauge,,microseconds,,Duration in microseconds to sync a single pod. Broken down by operation type: create, update, or sync,0,kubernetes,k8s.kubelet.pod.worker.duration
+kubernetes.kubelet.pod.worker.start.duration,gauge,,microseconds,,Duration in microseconds from seeing a pod to starting a worker,0,kubernetes,k8s.kubelet.pod.worker.start.duration
```

Comment:
microseconds is not a supported metric unit. It can be added https://github.com/DataDog/devops/wiki/Metrics-Metadata#uploadingupdating-catalog-metadata-for-an-integration (Step 7 specifically)

Reply:
Are you sure?
Because I found a lot of examples of `metadata.csv` which have metrics in `microseconds` like for ex.: https://github.com/DataDog/integrations-core/blob/master/cassandra/metadata.csv#L4
And I could find `microseconds` in the unit list here: https://github.com/DataDog/dogweb/blob/prod/integration/system/units_catalog.csv#L10

Reply:
Sorry, the CI was failing because `microseconds` (plural).